README.md

```diff
@@ -67,6 +81,23 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Russian](#russian)
 
 
+Icons and tags:
```

Comment:
There should be a link in the menu to a section called "Legend" or something... so that people can find easily what the icons/tags mean